### DISCUSSION TITLE
Compiling ROSCO with gfortran (compile errors)

### DISCUSSION THREAD

**Discussion by perri** [NatLabRockies/ROSCO#305] [Q&A] [answered]
Hi, I'm trying to compile ROSCO with gfortran, but get some compile errors. Are these bugs in the code?
Compiler output below:
`$ gfortran -c *.f90 -O3 -ffree-line-length-512 -w
ROSCO_IO.f90:910:40:

         IF (ABS(DebugOutData(I)) > 1E+99) THEN
                                        1
Error: Real constant overflows its kind at (1)
ROSCO_IO.f90:911:35:

             DebugOutData(I) = 1E+99
                                   1
Error: Real constant overflows its kind at (1)
ROSCO_IO.f90:912:11:

         END IF
           1
Error: Expecting END DO statement at (1)
ROSCO_IO.f90:919:43:

         IF (ABS(LocalVarOutData(I)) > 1E+99) THEN
                                           1
Error: Real constant overflows its kind at (1)
ROSCO_IO.f90:920:38:

             LocalVarOutData(I) = 1E+99
                                      1
Error: Real constant overflows its kind at (1)
ROSCO_IO.f90:921:11:

         END IF
           1
Error: Expecting END DO statement at (1)
`

**Comment by dzalkind** [comment]
It looks like your issue could be compiler specific, and it's not using `cmake` like we have previously set up.

Here are the latest installation instructions: https://rosco.readthedocs.io/en/latest/source/install.html. If you have issues with those, we would be happy to help.

Best, Dan

**Comment by perri** [comment]
Thank you for the answer. It seems, however, that there are no binaries provided for the latest release. It would be great if they were added.

**Comment by dzalkind** [comment, marked as answer]
The libraries have been added to the 2.9.0 Release page.

**Comment by perri** [comment]
Thank you!
